packages/graphql/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2.1'
+
+services:
+  mongo:
+    container_name: animavita-mongo
+    image: mongo
+    restart: always
+    ports:
+      - 27017:27017
+  localstack:
+    container_name: animavita-localstack
+    image: localstack/localstack
+    network_mode: bridge
+    ports:
+      - '4566:4566'
+      - '${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}'
+    environment:
```

Comment:
Just thinking out loud, should we consider having .env for these envs?

Reply:
I don't know... Thinking about docker-compose file, if someone needs to change some env it can be made directly here.